peachjam/views/save_document.py

```diff
@@ -142,8 +188,8 @@ def get_success_url(self):
         )
 
 
-class SavedDocumentCreateView(BaseSavedDocumentFormView, CreateView):
-    template_name = "peachjam/saved_document_create.html"
+class SavedDocumentCreateView(SavedDocumentFormMixin, CreateView):
```

Comment:
i noticed that we set the permission_required attribute but we forgot to add the permission required mixin for these views,

Reply:
good catch, added.